client/components/date-picker/events-tooltip.jsx

```diff
@@ -77,7 +77,7 @@ class EventsTooltip extends Component {
 								icon={ event.icon }
 								socialIcon={ event.socialIcon }
 								socialIconColor={ event.socialIconColor }
-								title={ event.title } />
+								title={ event.title === '' ? '(No title)' : event.title } />
 						</li>
```

Comment:
we should translate this string, Jeremery:

```es6
title={ event.title === '' ? this.props.translate( '(No title)' ) : event.title } />
```
should we add some style as well?, something like `<em>`?